service/src/NVMLIOGroup.cpp

```diff
@@ -94,7 +94,7 @@ namespace geopm
                                   Agg::sum,
                                   string_format_double
                                   }},
-                              {M_NAME_PREFIX + "MEMORY_FREQUENCY", {
```

Comment:
Why is this change in here?

Reply:
Because I haven't rewritten history yet.  This undoes a change in an earlier commit in this PR that started making changed in the NVML code.

I want to put that all in a separate PR.